lib/api.js

```diff
@@ -49,9 +49,8 @@ class API {
     this.skipFiles = config.skipFiles || [];
 
     this.log = config.log || console.log;
-
-    this.gasLimit = 0xffffffffff;            // default "gas sent" with transactions
-    this.gasLimitString = "0xfffffffffff";   // block gas limit for ganache (higher than "gas sent")
+    this.gasLimit = 0x1fffffffffffff;           // default "gas sent" with transactions
```

Comment:
```suggestion
    this.gasLimit = 0xffffffffff;           // default "gas sent" with transactions
```

Revert this change, possibly breaking. 

Double-check that we are using gasLimitString for HH's blockGasLimit.